applications/roloi-multisig-vault.md

```diff
@@ -0,0 +1,172 @@
+# Roloi Multi-Sig Vault
+
+- **Team Name:** NeoPower Digital
+- **Project Category:** Canary DApp
+- **Payment Address:** `1dz667uQX199rHyj6tizmL6EJe4AZxjH1RhnyrT1QuQ4pfS`
+
+## Project Overview :page_facing_up:
+### Overview
+
+Multi-Sig Vault Canary Dapp. 
+
+### Project Details
+
+**Roloi Multi-Sig Vault** is a generic multi-key wallet that empowers the security of funds. The DApp allows **sharing a vault between multiple users** to enable collective asset management.
```

Comment:
Could you compare your solution to the existing native multisig? What are the advantages and what are disadvantages? In what scenario this multisig is batter than the native one?